content/en/docs/tasks/administer-cluster/kms-provider.md

```diff
@@ -105,7 +174,14 @@ To encrypt the data:
          - secrets
        providers:
          - kms:
-             name: myKmsPlugin
+             apiVersion: v2
```

Comment:
@aramase is this right?  I thought it would be `v2alpha1`?

Reply:
https://github.com/kubernetes/kubernetes/pull/111126/files#diff-98f18019b1a1e4502439dc50f54de8977ca2ad2748b1502cea218d21033acf17R12

Reply:
The allowed values are `v2`, `v1` and default is `v1`